compiler/damlc/tests/daml-test-files/InterfaceSynonymTypeMismatchErrors.daml

```diff
@@ -0,0 +1,29 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module InterfaceSynonymTypeMismatchErrors where
+
+-- @ERROR range=27:7-27:24; Implementation of method ‘getValueBad’ on interface ‘I’ should return ‘Int’ but instead returns ‘Text’
```

Comment:
I wonder if this should use the actual name of the interface instead of or in addition to the synonym, but it wouldn't have to be in this PR

Reply:
Added to the backlog: https://github.com/digital-asset/daml/issues/16011